cargo-shuttle/README.md

```diff
@@ -0,0 +1,210 @@
+
+For example, running the following command will initialize a project for [rocket](https://rocket.rs/):
+
+```sh
+$ cargo shuttle init my-rocket-app
+```
+
+This should generate the following dependency in `Cargo.toml`:
+```toml
+shuttle-service = { version = "0.4.0", features = ["web-rocket"] }
+```
+
+The following boilerplate code should be generated into `src/lib.rs`:
+
+```rust
+#[macro_use]
+extern crate rocket;
+
+use shuttle_service::ShuttleRocket;
+
+#[get("/hello")]
+fn hello() -> &'static str {
+    "Hello, world!"
+}
+
+#[shuttle_service::main]
+async fn init() -> ShuttleRocket {
+    let rocket = rocket::build().mount("/", routes![hello]);
+
+    Ok(rocket)
+}
+```
+
+### Subcommand: `run`
+
+To run the shuttle project locally, use the following command:
+
+```sh
+# Inside your shuttle project
+$ cargo shuttle run
+```
+
+This will compile your shuttle project and start it on the default port `8000`. Test it by:
+
+```sh
+$ curl http://localhost:8000/hello
+Hello, world!
+```
+
+### Subcommand: `login`
+
+Use `cargo shuttle login` inside your shuttle project to generate an API key for the shuttle platform:
+
+```sh
+# Inside a shuttle project
+$ cargo shuttle login
+```
+
+This should automatically open a browser window with an auto-generated API key for your project. Simply copy-paste the API key back in your terminal or run the following command to complete login:
+
+```sh
+$ cargo shuttle login --api-key your-api-key-from-browser
+```
+
+### Subcommand: `deploy`
+
+To deploy your shuttle project to the cloud, run:
+
+```sh
+$ $ cargo shuttle deploy
+```
+
+Your service will immediately be available at `{crate_name}.shuttleapp.rs`. For instance:
+
+```sh
+$ curl https://my-rocket-app.shuttleapp.rs/hello
+Hello, world!
+```
+
+### Subcommand: `status`
+
+Check the status of your deployed shuttle project with:
+
+```sh
+$ cargo shuttle status
+```
+
+### Subcommand: `logs`
+
+Check the logs of your deployed shuttle project with:
+
+```sh
+$ cargo shuttle logs
+```
+
+### Subcommand: `auth`
+
+Run the following to create user credentials for shuttle platform:
+
+```sh
+$ cargo shuttle auth your-desired-username
+```
+
+### Subcommand: `delete`
+
+Once you are done with a deployment, you can delete it by running:
+
+```sh
+$ cargo shuttle delete
+```
+
+---
+
+<a id="development">
+<h1>Development</h1>
+</a>
+
+Thanks for using `cargo-shuttle`! We’re very happy to have you with us!
+
+During our alpha period, API keys are completely free and you can deploy as many services as you want.
+
+Just keep in mind that there may be some kinks that require us to take all deployments down once in a while. In certain circumstances we may also have to delete all the data associated with those deployments.
+
+🚀 To contribute to `cargo-shuttle` or stay updated with our development, please [open an issue, discussion or PR on Github](https://github.com/shuttle-hq/shuttle) and [join our Discord](https://discord.gg/H33rRDTm3p)!
```

Comment:
The rocket emoji looks a bit out of place at the beginning, imo. Can be left as is or be moved to the end of the sentence.